explorer/client/src/index.tsx

```diff
@@ -11,17 +11,22 @@ import './index.css'
 import * as serviceWorker from './serviceWorker'
 import { theme } from '@chainlink/styleguide'
 import ReactGA from 'react-ga'
-import { Config } from './config'
-
-ReactGA.initialize(Config.gaId())
-ReactGA.pageview(window.location.pathname + window.location.search)
+import * as api from './api/index'
 
 JavascriptTimeAgo.locale(en)
 moment.defaultFormat = 'YYYY-MM-DD h:mm:ss A'
 
 const muiTheme = createMuiTheme(theme)
 const store = createStore()
 
+async function setGa(): Promise<void> {
+  const { gaId } = await api.v1.config.getConfig()
```

Comment:
Hey @rupurt. Would you be able to tell me why i have an TS errors here?:
`Property 'gaId' does not exist on type 'ApiResponse<Config>'`

Also what do you think about moving config api call to the store?